src/components/add/ImgUploader.tsx

```diff
@@ -16,7 +16,7 @@ export const ImgUploader = forwardRef(
         css={imgInputCss}
         onChange={imgInputUploader}
         type="file"
-        accept="image/*, .jpg,.png,.bmp,.gif,.tif,.webp,.heic,.jpeg,.tiff,.heif"
+        accept="image/*, .jpg,.png,.bmp,.gif,.tif,.webp,.jpeg,.tiff,.heif"
```

Comment:
```suggestion
        accept="image/*, .jpg,.png,.bmp,.gif,.tif,.webp,.jpeg,.tiff"
```

Heif도 HEIC의 일종인 것 같더라고요! 요것도 사라져야 될 것 같아요~

Reply:
넵 수정해서 푸시할게요!